### PR TITLE
Mergify config file added

### LINKED ISSUE #127: Install and configure mergify

### PR DESCRIPTION
**Pull request by james02135**

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+pull_request_rules:
+  - name: Automatic merge ⬇️ on approval ✔
+    conditions:
+      - "#approved-reviews-by>=1"
```
Comment (abompard) [inline .mergify.yml]: This condition is not going to suffice, we want to make sure that unit tests pass before merging the command.
When a PR is created in Github, it does not have any checks. Then Github Actions or CentOS CI notices the PR and reports that CI is running. Only at that point will mergify notice that tests are not passing yet, but between the PR creation and the first CI status report there's a window where the PR seems to be correct (no failure). If somebody approves the PR during this window (which can be big if CI is slow that day), there's a risk of Mergify merging the PR before all tests have passed.
It is therefore recommended to list the required check that need to pass in the conditions.
See [Mergify's example](https://docs.mergify.io/examples/#automatic-merge-when-ci-works-and-approving-reviews) for this use case.

Comment (abompard) [reply .mergify.yml]: Also, I'd like to merge PRs only if the review(s) was done by someone in the team (which will be `@fedora-infra/data-team` in GitHub)

**Review by abompard — CHANGES_REQUESTED** [review]

**Comment by james02135** [comment]
pull_request_rules:
  - name: Automatic merge when CI passes and fedor-infra/data-team reviews
    conditions:
      - check-success=Run tests
      - approved-reviews-by=@fedora-infra/data-team
    actions:
      merge:
        method: merge

**Comment by james02135** [comment]
with GitHub Actions, the "check-success" condition only needs the name of the job, which in DN looks like "Run tests" if I'm not mistaken

and to ensure that a review is done by a "fedora-infra/data-team" member, it only needs the @ character

Does it look like there is anything I'm missing?

**Comment by abompard** [comment]
I think the mergify webapp has a feature where you can test that your config is valid, and how it would operate on a specific pull request in your repo. You can test it there, if necessary you can create a dummy pull request to test it with (make sure it's not going to merge something that has a failing test).

**Comment by james02135** [comment]
Used the mergify web app and confirmed that the config is valid and performed a test pull request and it worked perfectly

**Comment by abompard** [comment]
Could you rebase your branch on the `develop` branch please ? `git rebase develop`.

**Review by abompard — APPROVED** [review]

**Pull request merged**